### PR TITLE
Stopping large sized mails

### PR DESCRIPTION
**Pull request by Nada1996**
@smartinez87  We,"Ruby's Secret" team,basically wrote the code that first checks the size of the hash using object space and if te hash size is over 4000000 approx 4mb , then it loops only over the first 10 elements of the hash so only 10 lines would be printed

### CODE REVIEW

File: lib/exception_notifier/views/exception_notifier/exception_notification.html.erb [hunk 1]
```diff
@@ -45,7 +45,8 @@
         <tr>
           <td><%= raw summary %></td>
         </tr>
-      <% end %>
+        <% break if ObjectSpace.memsize_of(sections_content) > 4000000 && index == 9 %>
```
Comment (smartinez87) [inline lib/exception_notifier/views/exception_notifier/exception_notification.html.erb]: shouldn't this be 4*1024*1024 (4,194,304)?

**Comment by smartinez87** [comment]
hi @Nada1996, thanks for contributing!
Is it possible that you add a test case for this scenario?

File: lib/exception_notifier/views/exception_notifier/exception_notification.html.erb (hunk 1 above)
Comment (Nada1996) [inline lib/exception_notifier/views/exception_notifier/exception_notification.html.erb]: Yes, you're right , we first wrote just an approximate value to 4mb but I'll fix it right no to be exactly 4mb :+1:

**Comment by Nada1996** [comment]
@smartinez87 Yes sure,We're working on the test case :smiley:

**Pull request closed without merging**